keras_nlp/models/bert.py

```diff
@@ -391,17 +421,36 @@ def BertBase(weights=None, vocabulary_size=None, name=None, trainable=True):
 
     # TODO(jbischof): consider changing format from `h5` to
     # `tf.train.Checkpoint` once
-    # https://github.com/keras-team/keras/issues/16946 is resolved
-    if weights:
-        filepath = keras.utils.get_file(
-            "model.h5",
-            BASE_PATH + "bert_base_" + weights + "/model.h5",
-            cache_subdir="models/bert_base/" + weights + "/",
-            file_hash=checkpoints["bert_base"][weights]["md5"],
-        )
-        model.load_weights(filepath)
+    # https://github.com/keras-team/keras/issues/16946 is resolved. Applicable
+    # for other BERT variants as well.
+    if weights_filepath is not None:
+        model.load_weights(weights_filepath)
+
+    # TODO(jbischof): attach the tokenizer or create separate tokenizer class.
+    # Applicable for other BERT variants as well.
+    return model
+
+
+def BertLarge(weights=None, vocabulary_size=None, name=None, trainable=True):
+    weights_filepath, vocabulary_size = _handle_weights_and_vocab_size(
```

Comment:
I would just call this `weights`, to match the `load_weights` function. Conceptually you are getting the "updated" versions of both these parameters after checking for pretrained names. And at some point we may allow the input `weights` to be a filepath as well.